src/popups.rs

```diff
@@ -55,7 +55,7 @@ pub fn render_end_popup(frame: &mut Frame, sentence: String) {
         Line::from(sentence).alignment(Alignment::Center),
         Line::from(""),
         Line::from(""),
-        Line::from("Press R to start a new game").alignment(Alignment::Center),
+        Line::from("Press 'R' to start a new game").alignment(Alignment::Center),
```

Comment:
```suggestion
        Line::from("Press `R` to start a new game").alignment(Alignment::Center),
```

`` maybe to stay consistent with the rest of the modifications

Reply:
Well, it's true, but I thought that Esc and Space are special keys, therefore they got the \`\` sorrounding, while R is a simple character, and so I'd say it shall be sorrounded with ''.

But I might as well just change it to be consistent, which one do you prefer?

Reply:
it's really nitpicking but I prefer `` some people might say that R is also a key like `esc` or `space` :D